src/api/mock-interceptor.ts

```diff
@@ -92,17 +102,16 @@ export const interceptRequestsOnMock = () => {
   mock
     .onGet(new RegExp(`${API_PATHS.guiBackendAuthoritiesPath}/authorities/search.*`))
     .reply(200, mockAuthoritySearchResponse);
-  mock
-    .onGet(new RegExp(`${API_PATHS.guiBackendResourcesPath}/resources/resource-123/creators/.*/authorities`))
-    .reply((config) => loggedReply(config, 200, [mockCreatorOrContributorAuthoritiesResponse]));
-  mock.onGet(new RegExp(`${API_PATHS.guiBackendResourcesPath}/resources/.*/creators/.*/authorities`)).reply(200, []);
+
+  mockGetDelayedAndLogged(`${API_PATHS.guiBackendResourcesPath}/resources/resource-123/creators/.*/authorities`, 200, [
+    mockCreatorOrContributorAuthoritiesResponse,
+  ]);
+  mockGetDelayedAndLogged(`${API_PATHS.guiBackendResourcesPath}/resources/.*/creators/.*/authorities`, 200, []);
```

Comment:
Litt uenig at vi skal ha delay her, nå vil jo cypress kjøre enda saktere for min del. Tenker at det er unødvendig at de er her.

Jeg pleier å benytte setTimeout inni funksjonene som kaller api'et for å se hvordan ting fungerer når man venter på svar. (evt droppe å sette loading(false) i finally blokkene.

Reply:
cypress tenkte jeg ikke på, nei. Men om vi beholder funksjonene, men med delay som optional param,  vil det være mye lettere å endre ved behov